gazebo_plugins/test/worlds/gazebo_ros_wheelslip.world

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" ?>
+<!--
+  Gazebo ROS wheel slip plugin demo
+  Try for example:
+  To change slip compliance of a group of wheels:
+    ros2 param set /trisphere_cycle_slip0/wheel_slip_rear slip_compliance_unitless_longitudinal 1.0
+  To get the current slip compliance of a group of wheels:
+    ros2 param get /trisphere_cycle_slip0/wheel_slip_rear slip_compliance_unitless_longitudinal
+-->
+<sdf version="1.6">
+  <world name="default">
+    <gravity>-2 0 -9.8</gravity>
+
+    <include>
+      <uri>model://ground_plane</uri>
+    </include>
+    <include>
+      <uri>model://sun</uri>
+    </include>
+
+    <include>
+      <uri>model://trisphere_cycle</uri>
+      <name>trisphere_cycle_slip0</name>
+      <pose>0 0 0  0 0 0</pose>
+      <plugin name="wheel_slip_front" filename="libgazebo_ros_wheel_slip.so">
+
+        <ros>
+          <namespace>trisphere_cycle_slip0</namespace>
+          <parameter name="slip_compliance_parameters_global" value="1" type="bool"/>
```

Comment:
This doesn't appear to be a ROS parameter offered by the wheel slip plugin. Though, I'd be interested to know what happens if we set one of the global parameters here (ie. if it overrides the individual parameters or is ignored).